File: Javascript/Hashmap/E-448. Find All Numbers Disappeared in an Array.ts
function findDisappearedNumbers(nums: number[]): number[] {
  const set = new Set<number>(nums);
  const result: number[] = [];
  for (let i = 1; i <= nums.length; i++) {
    if (!set.has(i)) result.push(i);
  }
  return result;
}

File: Python/Hashmap/E-448. Find All Numbers Disappeared in an Array.py
class Solution:
    def findDisappearedNumbers(self, nums: list[int]) -> list[int]:
        nums_set = set(nums)
        result = []
        for i in range(1, len(nums) + 1):
            if i not in nums_set:
                result.append(i)

        return result
